src/components/MdComponents.tsx

```diff
@@ -32,44 +34,73 @@ import InfoBanner from "./InfoBanner"
 import Card from "./Card"
 import QuizWidget from "./Quiz/QuizWidget"
 import DocLink from "./DocLink"
+import Link from "@/components/Link"
+import { motion } from "framer-motion"
 // import Contributors from "@/components/Contributors"
 // import Logo from "@/components/Logo"
 // import MeetupList from "@/components/MeetupList"
 
 /**
  * Base HTML elements
  */
-export const commonHeadingProps: HeadingProps = {
+const headingPropsForAnchor = (id?: string): HeadingProps =>
+  id
+    ? ({
+        scrollMarginTop: 28,
```

Comment:
Got rid of false top margins and replaced with `scrollMarginTop`... fixes things like this:

https://github.com/ethereum/ethereum-org-fork/assets/54227730/b128f1d6-33aa-4f05-af72-03a4486e1bdc